pages/common/chsh.md

```diff
@@ -3,6 +3,14 @@
 > Change the user's login shell.
 > More information: <https://manned.org/chsh>.
 
-- Change the user's login shell:
+- Change your own login shell interactively:
 
-`chsh -s {{path/to/shell_binary}} {{username}}`
+`chsh`
+
+- Change the login shell of `user`:
+
+`chsh --shell {{/usr/bin/shell_binary}} {{user}}`
```

Comment:
```suggestion
`chsh --shell {{/usr/bin/shell}} {{user}}`
```

Reply:
Maybe a specific usage is better here like:
```suggestion
`chsh --shell {{/bin/zsh}} {{user}}`
```
or `chsh --shell $(which zsh)`

Reply:
I think it should be /usr/bin/zsh if we choice that.

Reply:
Why would it be?! Who has their shell in `/usr/bin`??

Reply:
Why assume that the shell is in `/usr/bin`? It is probably, but maybe not always.
```suggestion
`chsh --shell {{/path/to/shell}} {{username}}`
```

Reply:
It isn't on Darwin:

```
$ ls /usr/bin/zsh /usr/bin/bash /usr/bin/sh
ls: /usr/bin/bash: No such file or directory
ls: /usr/bin/sh: No such file or directory
ls: /usr/bin/zsh: No such file or directory
```

And probably on many Linux distros, too.

`/bin` is the standard directory for the shell, but I actually prefer

```suggestion
`chsh --shell $(which {{zsh}}) {{user}}`
```

Reply:
```
whereis zsh
zsh: /usr/bin/zsh /usr/lib/zsh /etc/zsh /usr/share/zsh /usr/share/man/man1/zsh.1.gz
```

Same applies for bash, zsh being installed from repos,

this is Arch

Reply:
Checking `chsh --list-shells` the path is `/bin/zsh`, also a lot of help posts in SO recommends `/bin/zsh`. Maybe it's the same result for that command.

Reply:
I think marchersimons suggestion is best, generic parameters are often the best. The diff between /usr/bin/zsh and /bin/zsh seems to be nothing major regarding permissions since both are equal

Reply:
And the same SHA hash

Reply:
`/usr/bin` tends to contain all custom shells, while `/bin` is reserved for system stuff:
```sh
❯ chsh --list-shells
/bin/sh
/bin/bash
/bin/zsh
/usr/bin/zsh
/usr/bin/git-shell
```

I would like to provide users a pointer where to find the shell if they are inexperienced, I would also be fine with `/bin` though